modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
+
+[id="jt-comparison-of-jenkins-openshift-pipelines-execution-models_{context}"]
+= Comparison of Jenkins and OpenShift Pipelines execution models
+
+Jenkins and OpenShift Pipelines offer similar functions but are different in architecture and execution. This section outlines a brief comparison of the two execution models.
```

Comment:
```suggestion
Jenkins and OpenShift Pipelines offer similar functions but are different in architecture and execution. 
```
Consider omitting the meta, pointing language that already stated in the heading that follows. I generally find that  phrasing like "in this section" and often the content that follows can nearly always be cut.

Reply:
Updated. Thanks.